### ISSUE TITLE
[Bug] AppImage v0.6.9 don't work

### ISSUE THREAD

**Issue by Rasterisko** [AppFlowy-IO/AppFlowy#6269]
### Bug Description

The version 0.6.9 appimage simply does not start.

### How to Reproduce

Download appimage v0.6.9, allow the program to run and execute.

### Expected Behavior

That the program would start

### Operating System

Linux Mint 21.2 XFCE

### AppFlowy Version(s)

0.6.9

### Additional Context

The version AppFlowy-0.6.9-linux-x86_64.deb works normally.

**Comment by ArnyminerZ** [comment]
Same for me on Ubuntu 22.04:
```
APPRUN_ERROR: No such file or directory
```

**Comment by s1nistr4** [comment]
Yeah, this should have extensive testing before releasing binaries, otherwise this wouldn't have happened.

**Comment by LucasXu0** [comment]
Sorry for the inconvenience! I'm still figuring out the issue and will release a new package once it has been addressed.

**Comment by LucasXu0** [comment]
It looks like this compatibility issue was caused by the update to Ubuntu 24.04 in the AppImage builder file.

https://github.com/AppFlowy-IO/AppFlowy/pull/5991

**Comment by LucasXu0** [comment]
I've uploaded the latest [AppImage](https://github.com/AppFlowy-IO/AppFlowy/releases/download/0.6.9/AppFlowy-0.6.9-linux-x86_64.AppImage) package which fixed the "No such file or directory" issue. Feel free to let me know if there are any further issues.

**Comment by dinucci** [comment]
It's not working here on my arch + KDE

Here's my output when executing ./AppFlowy-0.6.9-linux-x86_64.AppImage

![flowygl](https://github.com/user-attachments/assets/f9db7c71-93e9-4119-929c-6ebd6a381274)

inxi output:
[inxifxxxy.txt](https://github.com/user-attachments/files/17022664/inxifxxxy.txt)
